src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -87,6 +87,18 @@ public sealed class SystemConformanceProvider
             _urlResolver = urlResolver;
             _contextAccessor = contextAccessor;
             _searchParameterStatusManager = searchParameterStatusManager;
+
+            // Following code block will log same information multiple times due to Mediatr bug.
+            // Link for Mediatr bug https://github.com/jbogard/MediatR.Extensions.Microsoft.DependencyInjection/issues/34
+            if (!string.IsNullOrEmpty(_configuration.Value.Versioning.Default))
+            {
+                _logger.LogInformation($"Default version is:{_configuration.Value.Versioning.Default}.");
```

Comment:
Avoid string interpolation in log messages since we want the variables flow through.

```suggestion
                _logger.LogInformation($"Default version is: {VersioningDefault}.", _configuration.Value.Versioning.Default);
```

Reply:
Done with suggested changes. String interpolation is removed.